packages/rlc-common/src/metadata/packageJson/buildAzureStandalonePackage.ts

```diff
@@ -68,6 +68,8 @@ function getStandaloneCjsDevDependencies(config: AzurePackageInfoConfig) {
     "@rollup/plugin-json": "^6.0.0",
     "@rollup/plugin-multi-entry": "^6.0.0",
     "@rollup/plugin-node-resolve": "^13.1.3",
+    ...(config.moduleKind === "cjs" &&
```

Comment:
just confirm this dependency `cross-env` is only useful for cjs clients?

Reply:
Yes, ESM uses vitest and won't need cross-env to setup the TS_NODE_OPTIONS